batch/batch/utils.py

```diff
@@ -99,6 +99,12 @@ def adjust_cores_for_memory_request(cores_in_mcpu, memory_in_bytes, worker_type)
     return max(cores_in_mcpu, min_cores_mcpu)
 
 
+def adjust_cores_for_packability(cores_in_mcpu):
+    assert cores_in_mcpu != 0
```

Comment:
Two things:

Can we get some tests for this function?

We either need to rejects requests with 0 cpu or handle 0 here, otherwise this assertion will generate a 500.  The CPU regex doesn't allow negative values, so we're good there.

Reply:
We don't allow cores_mcpu to be 0 in the front_end, but I agree I should just handle it explicitly. I'll add tests just for this function.